aptos-move/framework/aptos-framework/sources/configs/gas_schedule.move

```diff
@@ -23,16 +31,27 @@ module aptos_framework::gas_schedule {
         entries: vector<GasEntry>
     }
 
+    /// Called during a module upgrade.
+    /// Creates and publishes the resources that are added at a later time.
+    public fun upgrade_module(aptos_framework: &signer) {
+        system_addresses::assert_aptos_framework(aptos_framework);
+
+        if (!exists<GasFeatureVersion>(signer::address_of(aptos_framework))) {
+            move_to<GasFeatureVersion>(aptos_framework, GasFeatureVersion { major: GENESIS_GAS_FEATURE_VERSION });
```

Comment:
nit: Why not just inline 0 instead of making it a constant if this is ever called once?

Reply:
Actually I think you are right. This must be 0 or otherwise we would enable the new behaviors as soon as we make this resource available, if GENESIS_GAS_FEATURE_VERSION is set to a current non-zero version.